test/SourceKit/CursorInfo/cursor_uses_swiftsourceinfo.swift

```diff
@@ -0,0 +1,20 @@
+// RUN: %empty-directory(%t/split)
+// RUN: %{python} %utils/split_file.py -o %t/split %s
+// RUN: %empty-directory(%t/build)
+// RUN: %target-swift-frontend -emit-module -module-name MyModule -emit-module-path %t/build/MyModule.swiftmodule -emit-module-source-info-path %t/build/MyModule.swiftsourceinfo %t/split/Action.swift
+// RUN: %sourcekitd-test -req=cursor -req-opts=retrieve_symbol_graph=1 -pos=5:14 %t/split/test.swift -- %t/split/test.swift -I %t/build | %FileCheck %s
```

Comment:
Any reason to include the symbol graph here rather than just doing cursor info on `Action` and checking the file/line/col? I would have thought we have a check for that already though, do we really not?

Reply:
Doing a cursor info on `Action` should also end up using the `.swiftsourceinfo`. `REFERENCED DECLS` does as well because the decl is actually the type of the decl you're doing cursor info on. But as I said, I'm very surprised we don't already have a test for that.

Reply:
For reasons I don’t fully understand (I haven’t really done any investigation though) we report the correct source location of `Action` even when we’re not using `.swiftsourceinfo`.